js/src/theme.js

```diff
@@ -213,10 +213,10 @@ const theme = createTheme({
         paddingRight: defaultTheme.spacing(4),
         paddingTop: defaultTheme.spacing(1.5),
         paddingBottom: defaultTheme.spacing(1.5),
-        [defaultTheme.breakpoints.down('sm')]: {
-          width: '100%',
-          borderRadius: 0,
-        },
+      },
+      sizeLarge: {
+        paddingTop: '12px',
```

Comment:
any reason to use fixed values instead of `theme.spacing()` helper?

Reply:
For internal button padding, I would keep static values. `theme.spacing` base (8px) can be changed at some point but when it will happen it shouldn't affect height-sensitive markup like navbar, buttons, icons, etc.

Reply:
thanks for the explanation @i-amdroid!

Reply:
done